pegasus/forms/hoc_signup_2018.rb

```diff
@@ -0,0 +1,14 @@
+require pegasus_dir 'forms/hoc_signup_2017'
+
+class HocSignup2018 < HocSignup2017
```

Comment:
No need to change anything, I'm just curious: why does each new year inherit from the previous year's form rather than from a shared generic template? This way we have to store all the previous years' forms...

Reply:
I wondered the same.  We could certainly modify `process_forms` to call on a generic handler for all `HocSignup` `kind` variants for for 2018 and later, but I'm not sure how to future-proof the code.  What's in this newly-added one is small, but still feels fairly specific.  Will keep pondering though.